frontend/admin/src/js/components/dashboard/Dashboard.tsx

```diff
@@ -1,5 +1,7 @@
 import React, { useRef } from "react";
 import { useIntl } from "react-intl";
+import { Bars3Icon } from "@heroicons/react/24/outline";
```

Comment:
Happy to see this kicked out of scope, but wondering how you feel about starting to alias all our icons to names that our semantic to our use for them? As in:

```suggestion
import { MenuIcon } from "@common/components/icons";
```